docs/api-ref.md

```diff
@@ -2686,7 +2685,7 @@ Source files: server/endpoint/schedules_endpoint.py
 #### schedule.create
 
 ```py
-schedule.create(schedule_item)
+schedules.create(schedule_item)
```

Comment:
For all 3 of these schedule-->schedules changes, could you also make them in the preceding heading? (Ex: `#### schedule.create`)

Reply:
Done